packages/camel-catalog/kaoto-camel-catalog-maven-plugin/src/test/java/io/kaoto/camelcatalog/CamelYamlDslSchemaProcessorTest.java

```diff
@@ -51,6 +51,18 @@ public void testProcessSubSchema() throws Exception {
         assertEquals("object", beanPropertySchema.get("type").asText());
     }
 
+    @Test
+    public void test_extract_single_oneOf_from_anyOf() throws Exception {
```

Comment:
maybe `testExtractSingleOneOfFromAnyOf()` - we rarely use C style underscore in Java